### DISCUSSION TITLE
Credit card icon to be added to Paste

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1257] [Request: Icon] [answered]
I need a generic credit icon for the Add Card pages within Billing console. If I could get the formatted SVG, the engineers on my team can add it to the Paste repo using the self-service instructions.

1. This is the icon that needs to be added. 
Streamline-Regular "Credit card 1"
![image](https://user-images.githubusercontent.com/72031986/110135401-64055600-7dc6-11eb-8393-715be899e9c1.png)

2. This is a mockup of the screen where it will be used. 
![image](https://user-images.githubusercontent.com/72031986/110135688-b6df0d80-7dc6-11eb-91cb-1dc1a83fdcea.png)

3. This will be a 'UI Icon'. 

Thank you!

**Comment by nkrantz** [comment]
Thanks for creating a discussion for this, @PoojMenon! We'll add the credit card icon to Figma and let you know once the SVG is ready for your eng team to add it to Paste.

**Reply by PoojMenon** [reply]
Thanks very much Nora!

**Comment by PoojMenon** [comment]
@serifluous Hi Sarah, saw an update on [DSYS-2652](https://issues.corp.twilio.com/browse/DSYS-2652). Could you let me know where I can find the icon svg?

**Reply by nkrantz** [reply]
Hi @PoojMenon - that ticket will be pulled into our next sprint. The update you saw might be the ticket's **AC** - acceptance criteria - which we use to signify when the ticket will be finished (we'll be ready to close out the ticket once the SVG file is ready and it's shared with your team). We'll make sure to let you know once it's ready ☺️

**Comment by serifluous** [comment, marked as answer]
PR submitted in #1267
